api/src/test/java/com/epam/pipeline/util/ControllerTestUtils.java

```diff
@@ -21,33 +21,130 @@
 import com.epam.pipeline.controller.Result;
 import com.fasterxml.jackson.core.type.TypeReference;
 import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.util.MultiValueMap;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
 
-public final class ControllerTestUtils {
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-    private ControllerTestUtils() {
-    }
+public class ControllerTestUtils {
+
+    private static final String SERVLET_PATH = "/restapi";
+    private static final String CONTENT_TYPE = "application/json;charset=UTF-8";
 
-    public static <T> ResponseResult<T> buildExpectedResult(final T payload) {
+    @Autowired
+    private JsonMapper jsonMapper;
```

Comment:
I would prefer to use constructor injection. (add `@RequiredArgsConstructor` and make fields public)